app/imports/api/methods.js

```diff
@@ -31,18 +31,11 @@ export const createApplication = new ValidatedMethod({
 
 // This method could be named better and maybe live
 // in a separate file.
-export const saveApplicationSection = new ValidatedMethod({
-  name: 'application.section.save',
+export const submitApplicationSection = new ValidatedMethod({
+  name: 'application.section.submit',
 
   validate({ step, data, applicationId }) {
```

Comment:
This only validates types. For `data` it only checks it an object.

Validation against schema happens inside `run()`.